### ISSUE TITLE
Quality booster decreases quality

### ISSUE THREAD

**Issue by MaluNoPeleke** [dvlden/uw-pub#35]
**Describe the bug**
If I enable the quality booster as a Pro user it decreases the quality instead of improving it.

**To Reproduce**
1. Go to ['...'](https://www.zdf.de/serien/der-schwarm/folge-7-200.html)
2. Enable quality booster
3. Play in fullscreen and watch for darker areas

**Expected behavior**
Normal colors/video quality

**Screenshots/Videos**
Quality booster enabled:
![image](https://github.com/user-attachments/assets/5bafa2c3-dd80-4c88-9b5e-53f9db1c487e)

Quality booster disabled:
![image](https://github.com/user-attachments/assets/e1902bc6-6674-48f5-9e03-2f816bfcd127)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser Vivaldi
 - Version 7

**Additional context**

**Comment by nnx0r** [comment]
Yes, well aware of that. As you can see, texture on the person is highly improved, but the blurry background or further distance that has darker colors is always screwed up.

I haven't be able to create a better solution so far, but I am open for suggestions from other developers.

<img width="509" alt="Screenshot 2024-11-27 at 10 31 52" src="https://github.com/user-attachments/assets/79c6a0b8-37ae-4932-8ca2-39b5cacd3550">

**Comment by nnx0r** [comment]
Just a note to everyone. In `v3.4.0` _(unreleased)_ and up, this feature is removed. 
There's no solution to fix color bendings in compositions, so this feature was more of a problem than a solution.

If someone disagrees and has used Quality Booster, upvote this comment and I'll bring it back with a note that it's problematic.

**Comment by nnx0r** [comment]
I did not receive any feedback, so this feature is removed in latest release. 
**In v3.4.1, this feature is no longer available.**
